File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
  if(arr2.length == arr1.length){
    const sumArr1 = arr1.reduce((acc, cur) => acc + cur)
    const sumArr2 = arr2.reduce((acc, cur) => acc + cur)
    return sumArr2 > sumArr1 ? -1 : sumArr2 === sumArr1 ? 0 : 1
  }
  return arr2.length > arr1.length ? -1 : 1
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public static int solution(int[] arr1, int[] arr2) {
        
        if (arr1.length > arr2.length) {
            return 1;
        } else if (arr1.length < arr2.length) {
            return -1;
        } else {
            
            int sum1 = 0;
            int sum2 = 0;
            
            for (int num : arr1) {
                sum1 += num;
            }
            for (int num : arr2) {
                sum2 += num;
            }
            
            if (sum1 > sum2) {
                return 1;
            } else if (sum1 < sum2) {
                return -1;
            } else {
                return 0;
            }
        }
    }
}